.github/workflows/macos.yml

```diff
@@ -110,6 +110,7 @@ jobs:
         with:
           ruby-version: ruby
           bundler-cache: true
+          cache-version: "bundler-cache-${{ matrix.postgresql-version }}"
```

Comment:
I think adding this prefix to the cache name will make it clear what is being cached.
For example, it will be useful when we see https://github.com/pgroonga/pgroonga/actions/caches here.

Reply:
`setup-ruby-bundler-cache-v6-macos-14-arm64-ruby-3.3.4-wd-/Users/runner/work/pgroonga/pgroonga-with--without--only--v-bundler-cache-16-Gemfile.lock-192197eaa669c740825adb55cc484952cfabe283954e8aa26d1c58ecfba28332`?

It includes `setup-ruby`. So it seems that we can distinguish the `setup-ruby` key and others without adding `bundler-cache` explicitly.

Reply:
You are right. I will remove this prefix because we don't need it by the time we will cache the other thing under `setup-ruby`.